src/Bridges/NetteDI/FilteredDIRepositoryFinder.php

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types = 1);
+
+namespace Nextras\Orm\Bridges\NetteDI;
+
+
+use Nette\DI\ContainerBuilder;
+use Nette\DI\Definitions\FactoryDefinition;
+use Nette\DI\Definitions\ServiceDefinition;
+use Nextras\Orm\Exception\InvalidStateException;
+use Nextras\Orm\Model\Model;
+use Nextras\Orm\Repository\IRepository;
+use ReflectionClass;
+
+
+class FilteredDIRepositoryFinder extends DIRepositoryFinder
```

Comment:
I'm not convinced this should be in Orm, it is a highly subjective use case. It would be ok to have it locally in your project, wouldn't it?

Reply:
Well, yes/maybe and no.

FilteredDIRepositoryFinder may be subjective and I don't insist this particular solution should be in Orm. I think however, that if Orm supports multiple solutions then it should also support auto-registering repositories that work with multiple solutions.

FilteredDIRepositoryFinder is just a proposal for a solution that works with minimal configuration because it reuses the model namespace.